src/css/markdown.scss

```diff
@@ -1,7 +1,14 @@
 div.markdown {
```

Comment:
I'd say generally speaking, the docs will always be more dense in nature vs. the marketing site and I think the content now causes more strain on the eyes to parse.

cc @ryanjwilke 

#### e.g
 
![Screenshot 2024-01-08 at 12 00 34 PM](https://github.com/cypress-io/cypress-documentation/assets/6298501/0d562f05-253d-4979-978c-6ba6dd2ac4cb)

Reply:
@jaffrepaul Can you be a bit more specific about the issues you're referring to here? I did notice that there's a difference in font sizes, but we are trying to make sure that the font-size remained at `18px` instead of `14px` which is what @elevatebart had last time I checked.

Reply:
Sure. I didn't realize uploading screenshot files in GH doesn't do you much good if you want to see them larger. 

To be more specific:

- Within the admonitions, the line height/whitespace now pushes the text all into one block https://share.cleanshot.com/S1bkZHYV

- Same here, just lost some space between the text & code block https://share.cleanshot.com/7D6c71bc

- Small space between the label & code block. This may be fine, just calling out what's definitively different since the global changes may affect other areas I didn't spot https://share.cleanshot.com/jC9zMk46

- In general, the more dense the copy, the more its a bit more squeezed together which makes it a bit harder to skim. https://share.cleanshot.com/jfR0JJ0B

Reply:
@jaffrepaul Cool. Thanks for the additional clarification. This is helpful. I agree with some of these points as well. Bart and I can sync on these soon.

Reply:
changed a few of these spacing issues spacing in the following commits e071b52, f039d4f, 00f09dc, c1403e3

Reply:
Cool. I think the only side affect that could use refinement is the uneven spacing in the adminitions. Particularly ones with an h2 https://share.cleanshot.com/JzlXYN87